Packs/CommonTypes/ReleaseNotes/3_1_1.md

```diff
@@ -0,0 +1,8 @@
+
+#### Incident Fields
+- **Username**
+- **Process Name**
+- **Process Path**
+- **Resource Type**
```

Comment:
to some of them you've just associated a new incident type, please mention that. e.g:
- Associated the **foo** field with the *bar* incident type.